yoda/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM java:8
+
+RUN apt-get update
+RUN apt-get install -yqq maven
+
+RUN mkdir -p /usr/jukebox/yoda
+WORKDIR /usr/jukebox/yoda
+
+COPY . /usr/jukebox/yoda
+
+RUN ["mvn", "dependency:resolve"]
+RUN ["mvn", "verify"]
+RUN ["mvn", "package"]
+
+ENTRYPOINT ["java", "-cp", "target/yoda-1.0-SNAPSHOT.jar", "life.jukebox.App"]
+
+ENV YOD_RPC_PORT 34000
```

Comment:
We don't need this env variable and the following expose line actually. Only need this once we actually start a gRPC server inside yoda that needs to listen to traffic on the network.